docs/.gitignore

```diff
@@ -1 +1,2 @@
 .hugo_build.lock
+.hvm
```

Comment:
I installed the hugo version manager to see if that would help me resolve issues with the links not working.